c4_set_parameters.m~

```diff
@@ -0,0 +1,95 @@
+% set_parameters: sets various parameters for the DLA detection
```

Comment:
I think this is a duplicate file generated by your editor and you might not want this in your codebase.

Reply:
Cleaned up!